react-elements/src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+const newH1 = React.createElement(
+  'h1',
+  [null],
```

Comment:
You want to pass `null` here. `null` tells React there are not any parameters. `[null]` tells React that there is one parameter and it is null. In this case, it happens to work, but in the general case this will cause problems.